src/storage/bloom/src/lib.rs

```diff
@@ -1,5 +1,9 @@
+// Copyright 2022 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
 //! This library contains an implementation of a bloom filter.
-//! 
+//!
```

Comment:
I'm surprised there's any formatting changes. Please make sure the cargo fmt test passes before merge.